Jellyfin.Api/Helpers/FileStreamResponseHelpers.cs

```diff
@@ -59,23 +51,13 @@ public static class FileStreamResponseHelpers
         /// </summary>
         /// <param name="path">The path to the file.</param>
         /// <param name="contentType">The content type of the file.</param>
-        /// <param name="isHeadRequest">Whether the current request is a HTTP HEAD request so only the headers get returned.</param>
         /// <param name="httpContext">The current http context.</param>
         /// <returns>An <see cref="ActionResult"/> the file.</returns>
         public static ActionResult GetStaticFileResult(
             string path,
             string contentType,
-            bool isHeadRequest,
             HttpContext httpContext)
```

Comment:
httpContext is no longer used, should be removed from the function

Reply:
Removed. VS Code isn't highlighting unused parameters for me. So that's weird.